i18n_scripts/split_translation_files.py

```diff
@@ -0,0 +1,121 @@
+import os, localizable
```

Comment:
```suggestion
"""
Write a high-level doc for this script.
"""


import os, localizable
```

Reply:
done

Reply:
@Amr-Nash please push the changes. I don't see it in the pull request.